README.rst

```diff
@@ -44,91 +45,151 @@ PennyLane supports Python 3.9 and above.
 .. header-end-inclusion-marker-do-not-remove
 
 
+********
 Features
-========
+********
+
+PennyLane-Lightning high performance simulators include the following backends:
+
+* ``lightning.qubit``: is a fast state-vector simulator written in C++.
+* ``lightning.gpu``: is a state-vector simulator based on the `NVIDIA cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_. It notably implements a distributed state-vector simulator based on MPI.
+* ``lightning.kokkos``: is a state-vector simulator written with `Kokkos <https://kokkos.github.io/kokkos-core-wiki/index.html>`_. It can exploit the inherent parallelism of modern processing units supporting the `OpenMP <https://www.openmp.org/>`_, `CUDA <https://developer.nvidia.com/cuda-toolkit>`_ or `HIP <https://docs.amd.com/projects/HIP/en/docs-5.3.0/index.html>`_ programming models.
+
+The following platforms are supported:
+
++-----------+---------+--------+-------------+----------------+-----------------+----------------+
+|           | L-Qubit | L-GPU  | L-GPU (MPI) | L-Kokkos (OMP) | L-Kokkos (CUDA) | L-Kokkos (HIP) |
++===========+=========+========+=============+================+=================+================+
+| Linux x86 | pip     | pip    | source      | pip            | source          | source         |
++-----------+---------+--------+-------------+----------------+-----------------+----------------+
+| Linux ARM | pip     | source |             | pip            | source          | source         |
++-----------+---------+--------+-------------+----------------+-----------------+----------------+
+| Linux PPC | pip     | source |             | pip            | source          | source         |
++-----------+---------+--------+-------------+----------------+-----------------+----------------+
+| MacOS x86 | pip     |        |             | pip            |                 |                |
++-----------+---------+--------+-------------+----------------+-----------------+----------------+
+| MacOS ARM | pip     |        |             | pip            |                 |                |
++-----------+---------+--------+-------------+----------------+-----------------+----------------+
+| Windows   | pip     |        |             |                |                 |                |
++-----------+---------+--------+-------------+----------------+-----------------+----------------+
```

Comment:
Useful table!

Reply:
We can update for conda/spack in a follow-up PR.